datasets/yahoo_answers_topics/README.md

```diff
@@ -0,0 +1,140 @@
+---
+annotations_creators:
+- found
+language_creators:
+- found
+languages:
+- en
+licenses:
+- unknown
+multilinguality:
+- monolingual
+size_categories:
+- n>1M
+source_datasets:
+- extended|other-yahoo-answers-corpus
```

Comment:
Are you still planning to add `yahoo-answers-corpus` as well during the sprint? If so should this just be `extended|yahoo-answers-corpus` so the tags will match up? cc @yjernite

Reply:
It's available [here](https://webscope.sandbox.yahoo.com/catalog.php?datatype=l) (L6 - Yahoo! Answers Comprehensive Questions and Answers version 1.0 ).
The website states that it's only available for academics and scientists, so maybe you are @yjernite  can request it.

BTW, added the yahoo_answers_qa in #998

Reply:
we can leave it this way and update when we add `yahoo-answers-corpus` ?